### PR TITLE
XS✔ ◾ Create rule.md - Choose correct timezone for your flows

### PR DESCRIPTION
**Pull request by amankumarrr**
## Reason for change (Issue, Email, conversation + reason, etc)
Adding a new rule to convert timezone when using Power Automate flows

### CODE REVIEW

**Review by bradystroud — APPROVED** [review]

File: rules/power-automate-flows-convert-timezone/rule.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+guid: f20c1d67-fbd5-4fa6-99ba-58b4c8f53fd7
+---
+
+When working with Power Automate flows, there are instances where you need to use or compare the time in your flow, by default, it is set to UTC.
+
+However, there are situations where using your local timezone might be more appropriate, especially for tasks or events that are specific to a particular region. It's important to choose the appropriate timezone based on the context and requirements of your task or system.
+
+<!--endintro-->
+
+Here is a bad example of comparing Datetime in default timezone in PA Flows:
```
Comment (tiagov8) [inline rules/power-automate-flows-convert-timezone/rule.md:19]: Hey @amankumarrr 

I don't think this sentence is needed.

File: rules/power-automate-flows-convert-timezone/rule.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+::: bad
+![Figure: Bad example - You are comparing the UTC with UTC](comparing-timedate-without-converting.png)
+:::
+
+- ❌ **Complex Time Zone Calculations**: Handling time zone conversions manually can become cumbersome, especially when dealing with events and meetings across multiple time zones.
```
Comment (tiagov8) [inline rules/power-automate-flows-convert-timezone/rule.md:25]: Hey @amankumarrr 

Can you follow https://www.ssw.com.au/rules/avoid-full-stops-in-bullet-point-lists in these list items?

File: rules/power-automate-flows-convert-timezone/rule.md (hunk 1 above)
Comment (amankumarrr) [reply rules/power-automate-flows-convert-timezone/rule.md:19]: removed it.

File: rules/power-automate-flows-convert-timezone/rule.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+When working with Power Automate flows, there are instances where you need to use or compare the time in your flow, by default, it is set to UTC.
+
+However, there are situations where using your local timezone might be more appropriate, especially for tasks or events that are specific to a particular region. It's important to choose the appropriate timezone based on the context and requirements of your task or system.
+
+<!--endintro-->
+
+Here is a bad example of comparing Datetime in default timezone in PA Flows:
+
+::: bad
+![Figure: Bad example - You are comparing the UTC with UTC](comparing-timedate-without-converting.png)
```
[the remark this answers is not in the document]
Comment (amankumarrr) [reply rules/power-automate-flows-convert-timezone/rule.md:22]: done

File: rules/power-automate-flows-convert-timezone/rule.md (hunk 2 above)
Comment (amankumarrr) [reply rules/power-automate-flows-convert-timezone/rule.md:25]: Done - removed

**Review by tiagov8 — APPROVED** [review]

**Review by bradystroud — APPROVED** [review]

**Pull request merged**